backend/app/helpers/spree/admin/navigation_helper.rb

```diff
@@ -225,26 +225,6 @@ def configurations_sidebar_menu_item(link_text, url, options = {})
           link_to(link_text, url, class: "#{'text-success' if is_selected} py-1 px-3 d-block sidebar-submenu-item")
         end
       end
-
-      def main_part_classes
```

Comment:
@MatthewKennedy let's not remove any methods, we should deprecate them first and remove later (eg spree 5.0)

Reply:
@damianlegawiec no problem. I'll fix this shortly.